griptape/drivers/embedding/voyageai_embedding_driver.py

```diff
@@ -8,7 +8,8 @@
 
 @define
 class VoyageAiEmbeddingDriver(BaseEmbeddingDriver):
-    """
+    """VoyageAI Embedding Driver.
```

Comment:
Not quite consistent with the other docstrings. For example the one for `AmazonS3FileManagerDriver` starts with a short sentence describing the classes purpose and does not split the class name.

Just pointing it out, but I'll approve anyway.

Reply:
Yeah it's not ideal, I just wanted to add _something_ to conform to the linter.